protovalidate/internal/rules.py

```diff
@@ -23,6 +23,9 @@
 from buf.validate import validate_pb2  # type: ignore
 from protovalidate.internal.cel_field_presence import InterpretedRunner, in_has
 
+# Convenience to stringify the type names for error messages
+FIELD_TYPE_NAMES = {v: k for k, v in vars(descriptor.FieldDescriptor).items() if k.startswith("TYPE_")}
```

Comment:
This seems rather ugly, I wonder if there's a better way to do it. (If not, it might be worth considering if we should just manually map out all of the types here instead of enumerating the variables under `FieldDesciptor`...) Anyway, not important enough to block on, just figured I'd call it out.

Reply:
Fair enough. I updated it to use a mapping in [b7e1cfa](https://github.com/bufbuild/protovalidate-python/pull/313/commits/b7e1cfaefca31617b70ba1a4205fca72e0068857). Lmk what you think.

Reply:
Yeah, I think that's fine. If we can think of any way to do this more elegantly we can always revisit it.